slycot/math.py

```diff
@@ -62,16 +63,16 @@ def mc01td(dico,dp,p):
         e.info = out[-1]
         raise e
     if out[-1] == 1:
-        warings.warn('entry P(x) is the zero polynomial.')
+        warnings.warn('entry P(x) is the zero polynomial.')
     if out[-1] == 2:
-        warings.warn('P(x) may have zeros very close to stability boundary.')
+        warnings.warn('P(x) may have zeros very close to stability boundary.')
     if out[-2] > 0:
-        warnings.warn('The degree of P(x) has been reduced to %i' %(dp-k))
+        warnings.warn('The degree of P(x) has been reduced to %i' %(dp-out[-2]))
     return out[:-2]
 
 
 def mb05md(a, delta, balanc='N'):
-    """Ar, Vr, Yr, VALRr, VALDr = mb05md(a, delta, balanc='N')
+    """Ar, Vr, Yr, VAL = mb05md(a, delta, balanc='N')
```

Comment:
Thanks for updating this.

This changes the API --- but, I'm not sure anyone is using it, in particular, python-control isn't using mb05md that I could see.

@repagh , it looks like you're the one who added this in 7b7eaad2 - do you know of users of this function?

Reply:
I have forgotten what I added it for....
Typically, this is used in calculating a discrete state-space from a continuous one. I see it is currently not used in python-control. Good idea to return the eigenvalue as a complex vector.